### PR TITLE
Remove VectorsWrapper around private Spark api

### PR DESCRIPTION
**Pull request by joshvfleming**
This replaces the wrapper with equivalent code.

### CODE REVIEW

File: photon-ml/src/main/scala/com/linkedin/photon/ml/util/VectorUtils.scala [hunk 1]
```diff
@@ -16,6 +16,7 @@ package com.linkedin.photon.ml.util
 
 
 import breeze.linalg.{DenseVector, SparseVector, Vector}
+import org.apache.spark.mllib.linalg.{DenseVector => SDV, SparseVector => SSV, Vector => SV}
```
Comment (ferd36) [inline photon-ml/src/main/scala/com/linkedin/photon/ml/util/VectorUtils.scala]: "SV"? In casual reading without seeing this import line, I would have thought it meant "SparseVector" already.

**Comment by joshvfleming** [comment]
@pogil No, but this is an emergency fix so that we have a workaround. I'll add tests next week.

**Comment by joshvfleming** [comment]
@XianXing They added a new rule to our internal byte-ray tool that blocked Azkaban uploads for any jar containing Spark package names. This PR was an emergency workaround to unblock our users.

Also, it's generally not a good practice to have dependencies on private APIs, because they can change at any time. The wrapper was essentially exposing a private Spark API. This change introduces a small amount of copy-paste, but it has the benefit of using only public APIs.

**Comment by XianXing** [comment]
Sounds reasonable @joshvfleming.

+ @TDDFT for more context.

**Comment by TDDFT** [comment]
LGTM

**Pull request merged**